.github/workflows/release-porting-guide.yml

```diff
@@ -0,0 +1,103 @@
+          Exact release version. For exmaple, 12.1.0
+        required: true
+
+jobs:
+  upload-porting-guide:
+    name: Extract the porting guide
+    runs-on: ubuntu-latest
+    env:
+      GIT_BRANCH: "release/porting-guide-${{ inputs.ansible-version }}"
+      CI_COMMIT_MESSAGE: >-
+        Add the Ansible community "${{ inputs.ansible-version }}" porting guide
+
+    steps:
+      - name: Compute derived context
+        id: context
+        env:
+          ANSIBLE_VERSION_FULL: ${{ inputs.ansible-version }}
+        run: >-
+          echo ansible-version-major="${ANSIBLE_VERSION_FULL//\.+([[:digit:]])}"
+          >> "${GITHUB_OUTPUT}"
+        shell: bash --noprofile --norc -O extglob -eEuo pipefail {0}
+
+      - name: Check out this repo src
+        uses: actions/checkout@v4
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/ansible-build-data
+          ref: ${{ inputs.ansible-build-data-branch }}
+          path: ansible-build-data
+
+      - name: Copy the RST file to the correct path
+        env:
+          ANSIBLE_VERSION_MAJOR: ${{ steps.context.outputs.ansible-version-major }}
+        run: >-
+          cp -v
+          "ansible-build-data/${ANSIBLE_VERSION_MAJOR}/porting_guide_${ANSIBLE_VERSION_MAJOR}.rst"
+          docs/docsite/rst/porting_guides/
+
+      - name: Set up git
+        run: |
+          git switch --create "${GIT_BRANCH}"
+          ACTOR_NAME="$(curl -s https://api.github.com/users/"${GITHUB_ACTOR}" | jq --raw-output '.name // .login')"
+          git config --global user.name "${ACTOR_NAME}"
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Add the porting guide
+        run: git add docs/docsite/rst/porting_guides/"${PORTING_GUIDE_RST}"
+
+      - name: Commit the porting guide
+        run: >-
+          git diff-index --quiet HEAD ||
+          git commit -m "${CI_COMMIT_MESSAGE}"
+
+      - name: Push to the repo
+        run: git push origin "${GIT_BRANCH}"
+
+      - name: Create the porting guide PR as draft
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          PR_BODY_MESSAGE: >-
+            ##### SUMMARY
+
+            This is a draft PR. Please mark the PR as ready for review to trigger PR checks.
+
+            ##### ISSUE TYPE
+
+            - Docs Pull Request
+
+            ##### COMPONENT NAME
+
+            docs/docsite/rst/porting_guides/porting_guide_${{ steps.context.outputs.ansible-version-major }}.rst
+        run: >-
+          gh pr create
+          --draft
+          --base devel
+          --head "release/porting-guide-${ANSIBLE_VERSION}"
+          --title "${CI_COMMIT_MESSAGE}"
+          --body "${PR_BODY_MESSAGE}"
+      - name: Retrieve the existing PR URL
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: >
+          echo -n pull_request_url= >> "${GITHUB_OUTPUT}"
```

Comment:
What is `pull_request_url` used for?

There used to be a job that somehow displayed it, I think:
```yaml
      - name: Log the pull request details
        run: >-
           echo 'PR URL: ${{ steps.pr.outputs.pull_request_url }}' | tee -a "${GITHUB_STEP_SUMMARY}"
```
but you removed it in b75dde88768391a42076a46a0b087cbb1de0a737.